docs/source/documentation/concepts.rst

```diff
@@ -76,9 +76,9 @@ Permissions for a user to download an asset
 
 Users of a organization can export (aka download) the following elements from Substra to their local environment:
 
-* the opener of a dataset if the organization has process permissions on the dataset
-* the archive of a function if the organization has process permissions on the function
-* the model outputted by a task if the organization has process permissions on the model and if this type of export has been enabled at deployment for the organization (environment variable model_export_enabled should be set to True)
+* The opener of a dataset if the organization has process permissions on the dataset.
+* The archive of a function if the organization has process permissions on the function.
+* The model outputted by a task if the organization has process permissions on the model **AND** if this type of export has been enabled at deployment for the organization. This means the environment variable ``model_export_enabled`` should be set to ``True`` in the organization's backend.
```

Comment:
```suggestion
* The model outputted by a task if the organization has process permissions on the model **AND** if this type of export has been enabled at deployment for the organization. This means the environment variable ``MODEL_EXPORT_ENABLED`` should be set to ``True`` in the organization's backend.
```
I'm used to see env variables in all caps, I don't know how much of a standard convention it is.

Reply:
Actually it's a helm value and not an environnement variable, my mistake. See here https://artifacthub.io/packages/helm/substra/substra-backend#orchestrator-settings.